inc/cachify.class.php

```diff
@@ -247,11 +247,8 @@ public function __construct() {
 			);
 
 			add_action(
-				'robots_txt',
-				array(
-					__CLASS__,
-					'robots_txt',
-				)
+				'do_robots',
+				'robots_txt'
```

Comment:
There is no public method `robots_txt`. Why has the reference to the static method been changed?
```php
array(
	__CLASS__,
	'robots_txt',
)